internal/Picker/Picker.module.less

```diff
@@ -26,6 +25,14 @@
 		margin: @agate-picker-margin;
 		padding: @agate-picker-padding;
 
+		.sizingPlaceholder {
+			height: 0;
+			visibility: hidden;
+			padding-left: .extract(@agate-picker-padding, left)[];
+			padding-right: .extract(@agate-picker-padding, right)[];
```

Comment:
![image](https://user-images.githubusercontent.com/4239873/94495843-c5306880-022d-11eb-878d-f40865d491fe.png)
The year text has no padding.

![image](https://user-images.githubusercontent.com/4239873/94495858-ceb9d080-022d-11eb-99cd-e4835fe6f85d.png)
But the `sizingPlaceholder` has the paddings. So the placeholder size will be bigger than the year text. Did you intend to do it?

Reply:
The width is the same as 90 + 12 +12 = 114.